src/platform/plugins/shared/esql/server/routes/lookup_index.ts

```diff
@@ -98,4 +98,55 @@ export const registerLookupIndexRoutes = (
       }
     }
   );
+
+  router.get(
+    {
+      path: '/internal/esql/lookup_index/privileges/{indexName?}',
```

Comment:
nit, putting the resource id directly after the resource name: 
```suggestion
      path: '/internal/esql/lookup_index/{indexName?}/privileges',
```

Reply:
that was my initial idea, but indexName is an optional param

Reply:
it could also be 2 separate routes, or a query param instead 🤔